Content.Shared/Atmos/EntitySystems/SharedGasMinerSystem.cs

```diff
@@ -0,0 +1,72 @@
+using Content.Shared.Atmos.Components;
+using Content.Shared.Examine;
+using Content.Shared.Temperature;
+
+namespace Content.Shared.Atmos.EntitySystems;
+
+public abstract class SharedGasMinerSystem : EntitySystem
+{
+    [Dependency] private readonly ExamineSystemShared _examineSystemShared = default!;
+    [Dependency] private readonly SharedAtmosphereSystem _sharedAtmosphereSystem = default!;
+
+    // Gas miners are traditionally placed in a sealed room behind many layers of glass windows
+    // and empty space. Under normal conditions a player is usually at least 5 tiles from
+    // a gas miner. The standard examine range (3) is too small for the gas miner's examine text
+    // to be really useful, so we have an extended range for examining gas miners.
+    private const float MinerExamineRange = 6f;
+
```

Comment:
I surveyed all the current maps to come to this conclusion. Six seems to be reasonable.